server/db.js

```diff
@@ -0,0 +1,58 @@
+const mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/abodely', { useNewUrlParser: true, useCreateIndex: true });
+
+const oneReviewSchema = new mongoose.Schema({
```

Comment:
A better name for this could be just `reviewSchema` and the other one will be `reviewsSchema`
Note the plural form of the word review.